ItemList/Source/ItemList.Database/Repositories/ItemsRepository.cs

```diff
@@ -1,31 +1,38 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Threading.Tasks;
+using ItemList.Contracts.Api;
 using ItemList.Contracts.Database;
 using ItemList.Contracts.Models;
+using MongoDB.Driver;
 
 namespace ItemList.Database.Repositories
 {
     public class ItemsRepository : IItemsRepository
     {
+        private readonly IMongoCollection<Item> _collection;
+
+        public ItemsRepository(IDatabaseConfiguration databaseConfiguration)
+        {
+            var client = new MongoClient(databaseConfiguration.GetConnectionString());
```

Comment:
Read more about `MongoClient` instance management in the official document. You use it as a request-scoped dependency which is not entirely correct.